tools/postinstall-fixup/linux-postinstall-fixup.sh

```diff
@@ -93,7 +106,7 @@ get-lib-deps-assoc() {
 get-lib-deps-assoc "$BUILD_DIR" "$INSTALL_DIR" "$OUTPUT_TMP"
 
 # Copy libs
-groups="libQt libpng libicu libmng libxcb libxkb libOpenGL"
+groups="libQt libpng libicu libmng libxcb libxkb libpcre2 libjbig libwebp libjpeg"
```

Comment:
removing libOpenGL, on purpose ?

Reply:
Yes, we need to backport https://github.com/sofa-framework/sofa/pull/1721 into master (which removes libOpenGL as it isn't working)

Reply:
Done in https://github.com/sofa-framework/sofa/pull/1729